apps/prairielearn/src/lib/config.js

```diff
@@ -50,6 +50,7 @@ const ConfigSchema = z.object({
   /** Overrides the user UIN in development with `authType: 'none'` */
   authUin: z.string().nullable().default('000000000'),
   authnCookieMaxAgeMilliseconds: z.number().default(30 * 24 * 60 * 60 * 1000),
+  sessionStoreExpireSeconds: z.number().default(84600),
```

Comment:
Nit, you probably meant this (one day):

```suggestion
  sessionStoreExpireSeconds: z.number().default(86400),
```